examples/outliers.ipynb

```diff
@@ -220,7 +220,7 @@
    "id": "9193f085",
```

Comment:
<p>You should also improve this text, add smth about that before the imputation step you firstly need to choose the appropriate imputation mehtod and you can do it using our EDA method for imputation visualisation  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>